src/clr-base/styles/_base-component.scss

```diff
@@ -0,0 +1,12 @@
+// Every Web Component will need to import a minimal amount of core CSS
+// This file should remain light as it is duplicated between components
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+:host,
+slot {
+  font-size: 20px;
```

Comment:
Not sure if this is staying or not but the base font-size can't be a pixel value.

Reply:
I can remove this, was just temporary to make sure base styles were applied.

Reply:
Removed